docker-compose.test.yml

```diff
@@ -9,3 +9,5 @@ services:
       - POSTGRES_HOST_AUTH_METHOD=trust
     ports:
       - "5432:5432"
+    volumes:
+      - ./database/schema.sql:/docker-entrypoint-initdb.d/init.sql
```

Comment:
Use the schema file to init the Postgres database followed [the document about Initialization scripts on Postgres docker image](https://hub.docker.com/_/postgres)